specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/ManagedEnvironmentsSecrets.json

```diff
@@ -0,0 +1,366 @@
+            "description": "Name of the Environment.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "name": "secretName",
+            "in": "path",
+            "description": "Name of the Secret.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ManagedEnvironmentSecret"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get an environment secret from an environment": {
+            "$ref": "./examples/ManagedEnvironmentSecrets_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagedEnvironmentSecrets"
+        ],
+        "summary": "Create or update a secret for a managedEnvironment.",
+        "description": "Create or update a secret for a managedEnvironment.",
+        "operationId": "ManagedEnvironmentSecrets_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Environment.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "name": "secretName",
+            "in": "path",
+            "description": "Name of the Secret.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "name": "secretEnvelope",
+            "in": "body",
+            "description": "properties of the secret",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedEnvironmentSecret"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ManagedEnvironmentSecret"
+            }
+          },
+          "201": {
```

Comment:
is it async operation? if yes, should markk as long running operation?

Reply:
no. 201 response is added to satisfy linter.